src/page/shipping-step.page.ts

```diff
@@ -6,7 +6,7 @@ export class ShippingStepPage {
 
   constructor () {
     this.tosRadioButton = $('#cgv');
-    this.checkoutButton = $('#form > p > button > span');
+    this.checkoutButton = $('#form > p > button');
```

Comment:
Try to find another selector, there is an attribute that makes this button unique